src/lua/InputApi.cpp

```diff
@@ -236,11 +236,7 @@ int LuaContext::input_api_is_mouse_button_released(lua_State* l) {
 int LuaContext::input_api_get_mouse_position(lua_State* l) {
 
   return LuaTools::exception_boundary_handle(l, [&] {
-    Point mouse_xy;
```

Comment:
Directly in this function yes. Otherwise this is a change of the Lua API. Which might be a good idea, but outside the scope of this pull request.